.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -124,15 +127,50 @@ function isObj(x: unknown): x is Record<string, unknown> {
   return typeof x === 'object' && x !== null;
 }
 
+interface FtrConfigsManifest {
+  defaultQueue?: string;
+  disabled?: string[];
+  enabled?: Array<string | { [configPath: string]: { queue: string } }>;
+}
+
 function getEnabledFtrConfigs(patterns?: string[]) {
+  const configs: {
+    enabled: Array<string | { [configPath: string]: { queue: string } }>;
+    defaultQueue: string | undefined;
+  } = { enabled: [], defaultQueue: undefined };
+  const uniqueQueues = new Set<string>();
+
+  for (const manifestRelPath of ALL_FTR_MANIFEST_REL_PATHS) {
+    try {
+      const ymlData = loadYaml(Fs.readFileSync(manifestRelPath, 'utf8'));
+      if (!isObj(ymlData)) {
+        throw new Error('expected yaml file to parse to an object');
+      }
+      const manifest = ymlData as FtrConfigsManifest;
+
+      configs.enabled.push(...(manifest?.enabled ?? []));
+      if (manifest.defaultQueue) {
+        uniqueQueues.add(manifest.defaultQueue);
+      }
+    } catch (_) {
+      const error = _ instanceof Error ? _ : new Error(`${_} thrown`);
```

Comment:
Personal happy (opposite of nit) is the `_`.
Years of fp makes me happy to see this Dima.  

No wasted time or thought in naming an uninteresting symbol.  Good stuff

Reply:
I was trying to make as little changes as possible and kept the original error handling by Brian in place, not sure about the code style that was followed here.

Reply:
Yeah, that's a notation style very common in fp langs, such as Haskell, F#.Net, etc.

Reply:
and in javascript